docs/UserGuide.md

```diff
@@ -16,171 +16,146 @@ By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbs
    > Having any Java 8 version is not enough. <br>
    > This app will not work with earlier versions of Java 8.
 
-1. Download the latest `addressbook.jar` from the [releases](../../../releases) tab.
-2. Copy the file to the folder you want to use as the home folder for your Address Book.
+1. Download the latest `taskmanager.jar` from the [releases](../../../releases) tab.
+2. Copy the file to the folder you want to use as the home folder for your Task Manager.
 3. Double-click the file to start the app. The GUI should appear in a few seconds.
-   > <img src="images/Ui.png" width="600">
-
 4. Type the command in the command box and press <kbd>Enter</kbd> to execute it. <br>
    e.g. typing **`help`** and pressing <kbd>Enter</kbd> will open the help window.
 5. Some example commands you can try:
-   * **`list`** : lists all contacts
-   * **`add`**` John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01` :
-     adds a contact named `John Doe` to the Address Book.
-   * **`delete`**` 3` : deletes the 3rd contact shown in the current list
+   * **`list`** : lists all tasks
+   * **`add`**` Finish Progress Report; 030217 1159PM` :
+     adds a task named `Finish Progress Report` with due date `3rd Feb 2017' and due time '1159PM` to the Task Manager.
+   * **`delete`**` project` : deletes the task with the name `project`
    * **`exit`** : exits the app
 6. Refer to the [Features](#features) section below for details of each command.<br>
 
 
 ## 2. Features
-
 > **Command Format**
 >
-> * Words in `UPPER_CASE` are the parameters.
-> * Items in `SQUARE_BRACKETS` are optional.
-> * Items with `...` after them can have multiple instances.
-> * Parameters can be in any order.
-
-### 2.1. Viewing help : `help`
+> * Duration must be specified in "hr", "min" and/or "sec".
+> * Time must be specified in "am" and/or "pm".
+> * For <... date and time> parameters, either date or time can be left out but not both.
+> * `<task>` refers to the name of the task.
+> * Parameters include: name, start time, start date, end (due) time, end (due) date, duration.
+
+### 2.1. Adding a task: `add`
+
+Adds a task to the task manager<br>
+Formats:
+> * `add <task>`
+> * `add <task>; <duration>`
+> * `add <task>; <due date and time>`
```

Comment:
can we change the order of the parameters? if so please specify.
if not, think how to improve usability on this aspect!